### ISSUE TITLE
Bug in print.experiment

### ISSUE THREAD

**Issue by choisy** [r-and-gama/rama#22]
The `print.experiment` methods returns an errors when there is only 1 parameter or 1 observed variable. Examples:
```
exp1 <- load_experiment("sir", system.file("examples", "sir.gaml", package = "rama"), "sir")
```
The `print.experiment` method works perfectly well on this object:
```
exp1
```
However, if I remove all the parameters but one:
```
exp2 <- exp1
for (i in 1:4) exp2[, 1] <- NULL
```
It doesn't work any more:
```
exp2
```
Same bug if I remove all the observed variables but one:
```
exp3 <- exp1
for (i in 1:2) exp3[, 6] <- NULL
exp3
```

**Comment by LucieContamin** [comment]
It also returns an error for NULL data frame. 
For example:
```r
exp1 <- load_experiment("sir", system.file("examples", "sir.gaml", package = "rama"), "sir")
exp2 <- exp1
exp2[, ] <- NULL
exp2
```

**Comment by choisy** [comment]
Fixed in 6cb5a800d85cd0f65c687f3c1f56ee04a550283a
